data/formats-data.js

```diff
@@ -5974,6 +5984,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	pikipek: {
+		randomBattleMoves: ["bravebird", "furyattack", "bulletseed", "brickbreak", "flamecharge", "uturn"],
```

Comment:
Fury attack maxes out at 75 bp, while return is 102 (return > fury attack)

Reply:
wow i mustve gotten mixed up with swipes or smth